src/main_chat/response_continuation.py

```diff
@@ -12,7 +12,7 @@ def __init__(self, db_proxy):
 
     def is_continuation_request_asked(self, input_statement):
         continuation_requests = set(self.db.get_responses_list_by_tags(tag="continue"))
-        sliced_statement = set(map(lambda x: SentenceFilter().extract_lemma(x), input_statement.split(' ')))
+        sliced_statement = set(map(lambda x: SentenceFilter().extract_lemma(x, True), input_statement.split(' ')))
```

Comment:
why it is `True`? is this value ever used in this method?

Reply:
it is used to make this line of code working due to changes in extract_lemma

Reply:
but it has a default value in `extract_lemma`, so I think this `True` parameter is not required here.

Reply:
yes, it has, but this default value was introduced to overload method extract_lemma. Here we must pass some argument(it may not be True xd) to distinguish different bahaviours of this method when calling from different places (extract_lemma is also used in filter_sentence_complex)

Reply:
I don't understand, which method does it override?

Reply:
as I wrote it overloads extract_lemma() method from SentenceFilter xd in Python there is no way to create new overloaded method with the same name but with different args(like in Java), so here it is overloading in Python